### PR TITLE
update code preview to look nicer

### PR DESCRIPTION
**Pull request by kentcdodds**
**What**: Change code preview to look nicer

**Why**: Because we care about looking nice

**How**: Pretty much copy all of styled-components 💅 

Here's some gifs:

responsive

preview

### CODE REVIEW

**Comment by kentcdodds** [comment]
The box shadow looks funny in those gifs, looks much better IRL:

screen shot 2017-07-08 at 10 10 38 pm

File: components/code-preview.js [hunk 1]
```diff
@@ -14,22 +14,42 @@ const StyledLiveProvider = glamorous(LiveProvider)({
...
 
 const StyledLivePreview = glamorous(LivePreview)({
   padding: '1rem',
+  flex: 1,
 })
 
-const StyledLiveError = glamorous(LiveError)((props, {colors, fonts}) => ({
-  color: colors.code,
+const StyledLiveError = glamorous(LiveError)(({theme: {fonts}}) => ({
+  color: 'white',
```
Comment (paulmolluzzo) [inline components/code-preview.js]: `colors.white` just in case.

File: components/code-preview.js [hunk 2]
```diff
@@ -14,22 +14,42 @@ const StyledLiveProvider = glamorous(LiveProvider)({
...
   fontFamily: fonts.monospace,
-  backgroundColor: colors.white,
-  flexBasis: '100%',
-  width: '100%',
-  maxWidth: '100%',
   padding: '1rem',
+  background: '#ff5555',
```
Comment (paulmolluzzo) [inline components/code-preview.js]: Should be in the theme, no?

File: components/code-preview.js [hunk 3]
```diff
@@ -14,22 +14,42 @@ const StyledLiveProvider = glamorous(LiveProvider)({
...
+  flex: 1,
+  display: 'block',
 }))
 
+const EditorContainer = glamorous.div(
+  {
+    display: 'flex',
+    justifyContent: 'stretch',
+    alignItems: 'stretch',
+    boxShadow: '0.05rem 0.05rem 1.12rem rgba(20, 20, 20, 0.27)',
```
Comment (paulmolluzzo) [inline components/code-preview.js]: Should be in the theme.

File: components/code-preview.js [hunk 4]
```diff
@@ -14,22 +14,42 @@ const StyledLiveProvider = glamorous(LiveProvider)({
...
 
-const StyledLiveEditor = glamorous(LiveEditor)({})
+const StyledLiveEditor = glamorous(LiveEditor)({flex: 1.7})
 
 const StyledLivePreview = glamorous(LivePreview)({
   padding: '1rem',
+  flex: 1,
 })
 
-const StyledLiveError = glamorous(LiveError)((props, {colors, fonts}) => ({
```
Comment (paulmolluzzo) [inline components/code-preview.js]: We should keep `colors`. See other comments.

File: components/code-preview.js (hunk 3 above)
Comment (paulmolluzzo) [inline components/code-preview.js]: To clarify, the entire `boxShadow` value should be in the theme. Sizes and color.

**Pull request merged**